testplan/common/report/base.py

```diff
@@ -53,7 +53,7 @@ def __init__(
         self.name = name
         self.description = description
 
-        self.uid = uid or uuid.uuid4()
+        self.uid = uid or str(uuid.uuid4())
```

Comment:
I don't like it that we need to add str() wherever we use uuid4(). Can we possibly remove it or extract it to a common util function?

Reply:
sure